src/rgw/rgw_sal_dbstore.h

```diff
@@ -232,20 +232,74 @@ class DBNotification : public Notification {
       friend class DBStore;
   };
 
+  class DBPlacementTier: public PlacementTier {
+    DBStore* store;
+    RGWZoneGroupPlacementTier tier;
+  public:
+    DBPlacementTier(DBStore* _store, const RGWZoneGroupPlacementTier& _tier) : store(_store), tier(_tier) {}
+    virtual ~DBPlacementTier() = default;
+
+    virtual const std::string& get_tier_type() { return tier.tier_type; }
+    virtual const std::string& get_storage_class() { return tier.storage_class; }
+    virtual bool retain_head_object() { return tier.retain_head_object; }
+    RGWZoneGroupPlacementTier& get_rt() { return tier; }
+  };
+
+  class DBZoneGroup : public ZoneGroup {
+    DBStore* store;
```

Comment:
similar to the comment above regarding `PlacementTier` vs. `RGWZoneGroupPlacementTier`, i don't think there's anything rados-specific in `RGWZoneGroup` that warrants wrapping this way. is this also just to hide the definition of `RGWZoneGroup` from rgw_sal.h for now?

Reply:
RGWZoneGroup is a RGWSystemMetaObj.  That is very rados specific.